proposals/0296-increase-tx-size.md

```diff
@@ -0,0 +1,177 @@
+
+- 2048 bytes or lower - 50% of bundles
+- 6144 bytes or lower = 65% of all bundles
+- 9216 bytes or lower = 100% of all bundles
+
+In consideration of the above, the use cases mentioned within the motivation
+section, and the current page size for managing memory within a validator, a
+new transaction size limit of 4096 bytes is proposed. This new limit should
+cover a majority of use cases mentioned within the motivation section, as well
+as enable most of the developers using Jito bundles for larger transactions to
+migrate to the new transaction format. Similarly, since the new limit proposed
+can accomodate the max accounts used in ALTs directly in the transaction,
+developers are able to migrate from `v0` transactions to `v1` transactions.
+
+A number of changes are required to be made to the validator to support the new
+transaction size limit. Consensus would need to be updated to support the
+larger transaction size, as larger transactions included in a block would be
+marked as invalid by the cluster today. The scheduler would also need to be
+modified to  support the larger transaction sizes. While this proposal does not
+introduce a new fee structure around bytes in a transaction, the scheduler
+should prioritize larger transactions differently, requiring developers to pay
+a higher priority fee to land their transaction.
+
+Testing for this new transaction size limit should be done extensively to
+ensure that performance on the cluster is not adversely affected. This testing
+should include the different use cases mentioned within the motivation section,
+as well as the different sizes of transactions that are currently being used by
+developers on the network.
+
+## Alternatives Considered
+
+Alternative designs considered:
+
+- Having a transaction loading feature that would allow developers to load
+the transaction parts in a buffer and then be able to execute them at the
+end.
+This method is no longer considered as it requires a much higher level of
+latency on the application level and a decent amount of complexity within
+the validator
+- Bundles at the protocol level. This would not solve all problems that are
+ solved by larger transaction sizes. Bundles would still limit the ability for
+ developers to use cryptographic signature schemes that have large proof sizes.
+
+## Impact
+
+Developers would have to update with applications to use the new transaction
+format to take advantage of the larger transaction size. Those developers that
+had previously been using address lookup tables would be required to update the
+new transactions with the full address list instead of the address lookup table
+and its indices.
+
+Over the long term, the new transaction format should require a migration to
+fully deprecate both `legacy` and `v0` transaction formats, and in deprecating
+`v0` transactions, the address lookup table feature would be removed from the
+protocol.
```

Comment:
Recommend removing this.

Deprecations like this are tractable when we can plausibly expect the elimination of old consumers from the ecosystem (as is the case with the natural drawdown of old validator versions) but the continued existence of dApps that rely on the `legacy` and `v0` transaction formats to function is assured. To remove support for a transaction format is to break every application that relies on those formats. The highlighted impact statement doesn't say that outright.